conda/core/solve.py

```diff
@@ -627,11 +661,12 @@ def _add_specs(self, ssc):
 
                 spec_set = (python_spec, ) + tuple(self.specs_to_add)
                 if ssc.r.get_conflicting_specs(spec_set):
-                    if self._repodata_fn == REPODATA_FN:
+                    if (self._repodata_fn == REPODATA_FN and
+                            ssc.update_modifier == UpdateModifier.UPDATE_SPECS):
```

Comment:
If the user passes a different update_modifier on the command line then this will never be triggered and a conflict list is never generated.

Reply:
For example:
```
conda create -n py27 python=2.7
...
conda install -n py27 --freeze-installed ipython=7
Collecting package metadata (current_repodata.json): done
Solving environment: failed with current_repodata.json, will retry with next repodata source.

UnsatisfiableError: 
```

Reply:
yep, needs a fix.  Thanks.